.teamcity/Gradle_Check/configurations/PerformanceTest.kt

```diff
@@ -40,7 +41,7 @@ class PerformanceTest(model: CIBuildModel, type: PerformanceTestType, stage: Sta
             tasks = ""
             gradleParams = (
                     gradleParameters(daemon = false)
-                    + listOf("clean distributed${type.taskId}s -x prepareSamples --baselines %performance.baselines% ${type.extraParameters} -PtimestampedVersion -Porg.gradle.performance.branchName=%teamcity.build.branch% -Porg.gradle.performance.db.url=%performance.db.url% -Porg.gradle.performance.db.username=%performance.db.username% -PteamCityUsername=%TC_USERNAME% -PteamCityPassword=%teamcity.password.restbot% -Porg.gradle.performance.buildTypeId=${IndividualPerformanceScenarioWorkers(model).id} -Porg.gradle.performance.workerTestTaskName=fullPerformanceTest -Porg.gradle.performance.coordinatorBuildId=%teamcity.build.id% -Porg.gradle.performance.db.password=%performance.db.password.tcagent%",
+                    + listOf("clean distributed${type.taskId}s -x prepareSamples ${baselinesParam} ${type.extraParameters} -PtimestampedVersion -Porg.gradle.performance.branchName=%teamcity.build.branch% -Porg.gradle.performance.db.url=%performance.db.url% -Porg.gradle.performance.db.username=%performance.db.username% -PteamCityUsername=%TC_USERNAME% -PteamCityPassword=%teamcity.password.restbot% -Porg.gradle.performance.buildTypeId=${IndividualPerformanceScenarioWorkers(model).id} -Porg.gradle.performance.workerTestTaskName=fullPerformanceTest -Porg.gradle.performance.coordinatorBuildId=%teamcity.build.id% -Porg.gradle.performance.db.password=%performance.db.password.tcagent%",
```

Comment:
Shouldn't we do that handling within the performance test task (i.e. on the Gradle build level)? I sometimes want to be able to pass an actual baseline to the performance test coordinator which would override whatever baseline is set. With this change, that would no longer be possible.

Reply:
Is there any way to:

- Set `--baselines xxx` when you specify this param on TC,
- Not pass `--baselines defaults` when you don't specify this param on TC,

?

Reply:
How about use empty string as "default" value when users do nothing? Is there any potential risk?